common/config/azure-pipelines/jobs/regression-testing.yaml

```diff
@@ -28,21 +28,21 @@ jobs:
 
     strategy:
       matrix:
-        "Windows_Node_20_9_0":
+        "Windows_Node_20_11_0":
```

Comment:
look at L5 of this file, it says "follows the tip of each subsequent major LTS (even-numbered) version", wouldnt we just need to use 20.x then?

Reply:
I guess we can, but that means future runs will never be in lockstep with the min supported version we have documented. We'd probably not have caught this regression if we passed in `20.x`, currently it would then use node `20.19`. I'd feel more comfortable removing the 2nd half of L5

We'd state 20.11.0 is our min supported, but

Reply:
Second half is left over from then we supported more than 2 Node.js major versions.

For example: in 4.11, this would run with Node 18.12 (minimum supported at the time), 20.x (middle one) and would **not** run with 22.x